pancloud/credentials.py

```diff
@@ -126,21 +120,21 @@ def refresh_token(self):
         return self.refresh_token_ or \
                self._resolve_credential('refresh_token')
 
-    def _fetch_credential(self, credential):
-        """Fetch credential from credentials file.
-
-        Args:
-            credential (str): Credential to fetch.
-
-        Returns:
-            str, None: Fetched credential or ``None``.
+    def _init_adapter(self):
+        module_path = self.adapter.rsplit('.', 1)[0]  # extracts 'pancloud.adapters' from 'pancloud.adapters.TinyDBStore'
```

Comment:
some long lines around here.  split comments.

Reply:
Those were temporary. I'll remove them from the codebase.